### PR TITLE
Create the suggestions architecture

### PR DESCRIPTION
**Pull request by alecslupu**
Fixes: #28

The suggestion page looks like: 
image

### CODE REVIEW

File: app/controllers/decidim/participatory_documents/admin/suggestions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        def filters
+          [
+            :state_eq
+          ]
+        end
+
+        def suggestions
+          # filtered_collection
+          paginate(filtered_collection)
```
Comment (microstudi) [inline app/controllers/decidim/participatory_documents/admin/suggestions_controller.rb:42]: I think that `filtered_collection` is already paginated.

image

**Review by microstudi — COMMENTED** [review]
Everything good, I found just a little thing. Can you check it?
Thanks!

File: app/controllers/decidim/participatory_documents/admin/suggestions_controller.rb (hunk 1 above)
Comment (alecslupu) [reply app/controllers/decidim/participatory_documents/admin/suggestions_controller.rb]: Yes, it is. We are not overriding filtered_collection, therefore, i have removed the pagination call.

**Comment by microstudi** [comment]
Looks like some test is failing

**Comment by alecslupu** [comment]
> Looks like some test is failing

All green!

**Review by microstudi — APPROVED** [review]

**Pull request merged**